.github/workflows/coverage.yml

```diff
@@ -15,8 +16,7 @@ jobs:
   coverageReport:
     strategy:
       matrix:
-        api-level: [21,30]
-        target: [ google_apis ]
```

Comment:
What's wrong with `google_apis` ? IMO There are better than default APIs.

Reply:
hi @gouri-panda ,
IMO google_apis is not better then default api.
when i try to run tests on google_apis then android 30 start failing with android 33.
also in nightly we are using default api and there test are working fine.

Reply:
```google_apis``` are much heavier then default API, its required more high system configuration then default that's why its failing on github runner while works on local. UI screen started sending ANR signal leads to test failure.

Reply:
I debug the nightly build its running on ```default_apis``` but labelled as ```google_apis```, I created a ticket for it. https://github.com/kiwix/kiwix-android/issues/3095

Reply:
Google_api are not free/foss either AFAIK, we should not rely on them.